pkgs/by-name/xt/xtf/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchgit,
+  stdenv,
+  doxygen,
+  python3Packages,
+}:
+
+stdenv.mkDerivation {
+  pname = "xtf";
+  version = "0-unstable-2024-07-25";
+
+  outputs = [
+    "out" # xtf-runner and test suite.
+    "doc" # Autogenerated HTML documentation website.
+    "dev" # Development headers.
+  ];
+
+  src = fetchgit {
+    url = "https://xenbits.xenproject.org/git-http/xtf.git";
+    rev = "f37c4574dd79d058c035be989ac6648508556a1a";
+    hash = "sha256-3eOKQXdyFX0iY90UruK8lLfnXQt+cOlvyW/KMj2hczQ=";
+  };
+
+  nativeBuildInputs =
+    (with python3Packages; [
+      python
+      wrapPython
+    ])
+    ++ [ doxygen ];
+
+  buildFlags = [ "doxygen" ];
+
+  # The Makefile writes the finished objects to $DESTDIR$xtfdir.
+  installFlags = [
+    "DESTDIR=$(out)"
+    "xtfdir=/share/xtf"
+  ];
+
+  postInstall =
+    # Copy autogenerated documentation to $doc.
+    ''
+      mkdir -p $doc/share/doc/xtf
```

Use the more general variables:

```suggestion
      mkdir -p ''${!outputDoc}/share/doc/xtf
```